src/components/modalComponent.jsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const ModalWrapper = styled.div`
+  position: fixed;
+  z-index: 1000;
+  background-color: rgba(0, 0, 0, 0.8);
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+`;
+
+const OuterModal = styled.div`
+  position: absolute;
+  box-sizing: border-box;
+  box-shadow: 5px 5px 10px black;
+  background-color: rgba(138, 214, 204, 0.9);
+  border-radius: 10px;
+  top: calc(20vh);
+  left: calc(30vw);
+  width: 520px;
+  height: 480px;
+  padding: 40px 20px;
+  margin: -50px;
+  z-index:100;
```

Comment:
```suggestion
  z-index: 100;
```

스타일린트가 작동을 잘 안하는 것 같네요. 이 부분 수정해서 PR 만들어야겠습니다🥲

Reply:
넵... 저 다른 부분에서 trailing commas 가 잡히지 않은 것을 발견했습니다...!

Reply:
해당 이슈 PR 머지되었습니다! 최근에 작업된 부분이라 이후에 dev 브랜치 pull 받으실 때에는 적용되어 있을 거예요! 확인 감사합니다😃